CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ For more information and how-to, see [RFC: Keep A Changelog](https://github.com/
 ### Fixed
 
 - Update description of memory stressors [#3225](https://github.com/chaos-mesh/chaos-mesh/pull/3225)
+- Refactor StaticCheck for the code quality [#3227](https://github.com/chaos-mesh/chaos-mesh/pull/3227)
```

Comment:
I think we'd better move this entry to the bottom of section `Changed`.

Although this activity is called "bug bash", they are not actually "bugs" :) Most of them are code bad-smell which might introduce potential issues.

Reply:
rest LGTM :D

Reply:
and maybe use more detailed words to describe what you changed.

Reply:
Sure! will do that.